### PR TITLE
Add SunHelm 3rd party patch messages

### PR DESCRIPTION
**Pull request by sean-kang**

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -9214,6 +9214,76 @@ plugins:
...
+      - <<: *patch3rdParty
+        subs:
+          - 'Dugongs and Manatees - Mihail Monsters and Animals'
+          - '[SunHelm Survival - Compatibility Patches](https://www.nexusmods.com/skyrimspecialedition/mods/41335/)'
+        condition: 'active("mihailseacow.esp") and not active("SunHelm_mihailseacow.esp")'
+      - <<: *patch3rdParty
+        subs:
+          - 'Skyrim Alchemy and Food Overhaul'
+          - '[SunHelm Survival - Compatibility Patches](https://www.nexusmods.com/skyrimspecialedition/mods/41335/)'
+        condition: 'active("SAFO.esp") and not active("SunHelm_SAFO.esp")'
```
Comment (MacSplody) [inline masterlist.yaml]:
```suggestion
      - <<: *patch3rdParty
        subs:
          - 'Clockwork'
          - '[SunHelm Survival - Compatibility Patches](https://www.nexusmods.com/skyrimspecialedition/mods/41335/)'
        condition: 'active("Clockwork.esp") and not active("SunHelm_Clockwork.esp")'
      - <<: *patch3rdParty
        subs:
          - 'Dawn of Skyrim (Director''s Cut) SE 1.4'
          - '[SunHelm Survival - Compatibility Patches](https://www.nexusmods.com/skyrimspecialedition/mods/41335/)'
        condition: 'active("Blues Skyrim.esp") and not active("SunHelm_BluesSkyrim.esp")'
      - <<: *patch3rdParty
        subs:
          - 'Dugongs and Manatees - Mihail Monsters and Animals'
          - '[SunHelm Survival - Compatibility Patches](https://www.nexusmods.com/skyrimspecialedition/mods/41335/)'
        condition: 'active("mihailseacow.esp") and not active("SunHelm_mihailseacow.esp")'
      - <<: *patch3rdParty
        subs:
          - 'Falskaar'
          - '[SunHelm Survival - Compatibility Patches](https://www.nexusmods.com/skyrimspecialedition/mods/41335/)'
        condition: 'active("Falskaar.esm") and not active("SunHelm_Falskaar.esp")'
      - <<: *patch3rdParty
        subs:
          - 'INIGO'
          - '[SunHelm Survival - Compatibility Patches](https://www.nexusmods.com/skyrimspecialedition/mods/41335/)'
        condition: 'active("Inigo.esp") and not active("SunHelm_Inigo.esp")'
      - <<: *patch3rdParty
        subs:
          - 'Interesting NPCs'
          - '[SunHelm Survival - Compatibility Patches](https://www.nexusmods.com/skyrimspecialedition/mods/41335/)'
        condition: 'active("3DNPC.esp") and not active("SunHelm_3DNPC.esp")'
      - <<: *patch3rdParty
        subs:
          - 'Moonpath to Elsweyr SSE'
          - '[SunHelm Survival - Compatibility Patches](https://www.nexusmods.com/skyrimspecialedition/mods/41335/)'
        condition: 'active("moonpath.esp") and not active("SunHelm_moonpath.esp")'
      - <<: *patch3rdParty
        subs:
          - 'Project AHO'
          - '[SunHelm Survival - Compatibility Patches](https://www.nexusmods.com/skyrimspecialedition/mods/41335/)'
        condition: 'active("Dwarfsphere.esp") and not active("SunHelm_Dwarfsphere.esp")'
      - <<: *patch3rdParty
        subs:
          - 'Riften Docks Overhaul'
          - '[SunHelm Survival - Compatibility Patches](https://www.nexusmods.com/skyrimspecialedition/mods/41335/)'
        condition: 'active("Riften Docks Overhaul.esp") and not active("SunHelm_RiftenDocksOverhaul.esp")'
      - <<: *patch3rdParty
        subs:
          - 'Skyrim Alchemy and Food Overhaul'
          - '[SunHelm Survival - Compatibility Patches](https://www.nexusmods.com/skyrimspecialedition/mods/41335/)'
        condition: 'active("SAFO.esp") and not active("SunHelm_SAFO.esp")'
      - <<: *patch3rdParty
        subs:
          - 'Solitude Expansion'
          - '[SunHelm Survival - Compatibility Patches](https://www.nexusmods.com/skyrimspecialedition/mods/41335/)'
        condition: 'active("Solitude Expansion.esp") and not active("SunHelm_SolitudeExpansion.esp")'
      - <<: *patch3rdParty
        subs:
          - 'The Forgotten City'
          - '[SunHelm Survival - Compatibility Patches](https://www.nexusmods.com/skyrimspecialedition/mods/41335/)'
        condition: 'active("ForgottenCity.esp") and not active("SunHelm_ForgottenCity.esp")'
      - <<: *patch3rdParty
        subs:
          - 'Wintersun - Faiths of Skyrim'
          - '[SunHelm Survival - Compatibility Patches](https://www.nexusmods.com/skyrimspecialedition/mods/41335/)'
        condition: 'active("Wintersun - Faiths of Skyrim.esp") and not active("SunHelm_Wintersun.esp")'
      - <<: *patch3rdParty
        subs:
          - 'Wyrmstooth'
          - '[SunHelm Survival - Compatibility Patches](https://www.nexusmods.com/skyrimspecialedition/mods/41335/)'
        condition: 'active("Wyrmstooth.esp") and not active("SunHelm_Wyrmstooth.esp")'
```

**Review by MacSplody — CHANGES_REQUESTED** [review]
Just need to reorder the messages by the mods name.

**Comment by sean-kang** [comment]
Thanks for the update.

**Review by MacSplody — APPROVED** [review]

**Comment by MacSplody** [comment]
Thanks.

**Pull request merged**